src/main/scala/nl/biopet/utils/ngs/vcf/SampleCompare.scala

```diff
@@ -178,6 +178,42 @@ class SampleCompare(header: VCFHeader) extends Serializable {
                              relative: Boolean = false): Unit =
     writeOverlapFile(refAllelesCounts, outputFile, relative)
 
+  /**
+    * Writes non reference genotype matrix to a file
+    * @param outputFile File to write to
+    * @param relative If true values will be devided by the total number of counts for a sample
+    */
+  def writeNonRefGenotypeOverlap(outputFile: File,
+                                 relative: Boolean = false): Unit = {
+    writeOverlapFile(substractMatrix(genotypesCounts, refGenotypesCounts),
+                     outputFile,
+                     relative)
+  }
+
+  /**
+    * Writes non reference allele matrix to a file
+    * @param outputFile File to write to
+    * @param relative If true values will be devided by the total number of counts for a sample
+    */
+  def writeNonRefAllelesOverlap(outputFile: File,
+                                relative: Boolean = false): Unit = {
+    writeOverlapFile(substractMatrix(allelesCounts, refAllelesCounts),
+                     outputFile,
+                     relative)
+  }
+
+  /** Subtract matrix from an other */
+  def substractMatrix(start: Array[Array[Long]],
+                      substract: Array[Array[Long]]): Array[Array[Long]] = {
```

Comment:
`substract` sounds like a method, so it confused me slightly when reading the code below. But I also cannot think of a better name. Just putting this out there.